force-app/main/default/lwc/rd2ChangeEntry/rd2ChangeEntry.html

```diff
@@ -0,0 +1,22 @@
+<template>
+    <lightning-layout vertical-align="center">
+        <lightning-layout-item size="4" class="slds-p-around_x-small">
+            <b><lightning-formatted-date-time value={changeEntry.changeDate}></lightning-formatted-date-time></b>
+        </lightning-layout-item>
+        <lightning-layout-item size="8">
+            <lightning-badge label={changeEntry.changeType} class={badgeClass}></lightning-badge>
+        </lightning-layout-item>
+    </lightning-layout>
+    <template for:each={changeEntry.fields} for:item='item'>
```

Comment:
Could we sync the naming here with fields and items? since the `for:item` here is already "item" and the child component name is `rd2ChangeEntryItem` it might make the most sense to change the `changeEntry` property from `fields` to something like `entryItems` and then the for:item could also be changed to `entryItem` so that it all matches.